Nginx_Auth_Proxy/test/nginx-auth-proxy.test.js

```diff
@@ -7,15 +7,19 @@ const axios = axiosClass.create({
     rejectUnauthorized: false,
   }),
 });
+
 const axiosConfig = { validateStatus: false };
+
 const mockStatus = (status) => (ctx) => {
   ctx.status = status;
 };
+
 const mockResponse = (status, body) => (ctx) => {
   ctx.status = status;
   ctx.body = body;
 };
-const testHost = process.env.TEST_HOST || "localhost:6443";
+
+const TEST_HOSTS = process.env.TEST_HOSTS.split(",") || ["https://localhost:6443", "http://localhost:8080"];
```

Comment:
(minor nitpick but feel free to ignore). Consistency wise, previously we had global constant variables marked with cammel case not full upper case. Can ignore though, not a show stopper